portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,21 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body class="site-background">
+    <div class="header">
+      <h1>Hi, I'm Shreya.</h1>
     </div>
+    <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a onclick="showAbout()">About Me</a></li>
+        <li><a onclick="showPics()">Random Picture from my Life!</a></li>
+        <li><a href="linkedin">LinkedIn</a></li>
+    </ul>
+    <div id="aboutSpace">
+      <p class="about-hide"> I'm a rising junior at UCLA, majoring in Linguistics and Computer Science (that's one major)! In my free time, I enjoy reading and playing the piano.  Recently, I've started learning to cook (I'm going to need this for my college apartment next year) and playing Stardew Valley (turns out video games are a great way to stay connected to friends during this time)!</p>
```

Comment:
break this line up so it's multiple lines, each line less than 100 characters
additionally, indent the lines after the first line in by 2